src/app/item-popup/ItemPopupHeader.scss

```diff
@@ -106,15 +108,38 @@ $item-header-spacing: 5px;
     vertical-align: top;
     width: 16px;
     background-size: contain;
+    filter: drop-shadow(0px 0px 1px $icon-bg-shadow) drop-shadow(0px 0px 0px $icon-bg-shadow);
```

Comment:
What's the second shadow for? With a 0px spread I'm not clear on what it'd do.

Reply:
this layers in a slightly harder inner edge to supplement the blurred part of the shadow